content/en/docs/ops/best-practices/deployment/index.md

```diff
@@ -1,9 +1,8 @@
 ---
 title: Deployment Best Practices
-description: General best practices for your Istio deployments.
-weight: 2
-icon: best-practices
-keywords: [deployment-models, cluster, availability-zones, control-plane]
+description: General best practices when setting up an Istio service mesh.
+force_inline_toc: true
```

Comment:
Did you intend to have force_inline_toc on the best practices pages?

Reply:
Yes, I thought it might be good to have a list of the best practices at the top. WDYT?